utils/RequestResponse.go

```diff
@@ -6,7 +6,7 @@ type UpdateFeatureFlagRequest struct {
 }
 
 type CreateFeatureFlagRequest struct {
-	FlagName    string `json:"Name" validate:"required,unique"`
+	FlagName    string `json:"Name" validate:"required"`
```

Comment:
why did we remove unique

Reply:
Because unique is only to check in case of slice if the entry is unique in a set of entries.
REF : https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Unique

Reply:
oh ok, so it isn't related to storing a unique id in the DB?

Reply:
Yes it isn't, resolving this comment. @RitikJaiswal75